tsconfig.json

```diff
@@ -1,17 +1,14 @@
 {
+  "extends": "@tsconfig/node22/tsconfig.json",
   "compileOnSave": true,
   "compilerOptions": {
-    "target": "es2018",
-    "module": "commonjs",
-    "moduleResolution": "node",
     "outDir": "./dist",
     "sourceMap": true,
-    "skipLibCheck": true,
     "noEmit": false,
+    "strict": false,
     "allowJs": true,
     "checkJs": true,
     "resolveJsonModule": true,
-    "esModuleInterop": true,
```

Comment:
I'm guessing the removed properties appear in `@tsconfig/node22/tsconfig.json`?

Reply:
Yes. https://www.npmjs.com/package/@tsconfig/node22?activeTab=code

Reply:
the new `strict` property is there to override the new default which is `true` without which our linting is very very sad.